packages/web-runtime/src/components/Topbar/NotificationBell.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <oc-button
+    id="oc-notification-bell"
+    v-oc-tooltip="notificationsLabel"
+    appearance="raw"
+    variation="inverse"
+    :aria-label="notificationsLabel"
+  >
+    <oc-icon
+      class="oc-cursor-pointer oc-flex oc-flex-middle"
+      name="notification-3"
+      fill-type="line"
+    />
+    <span
+      v-if="notificationCount"
+      :key="notificationCount"
+      :class="{ shake: animate, badge: true }"
+      v-text="notificationCountLabel"
+    />
+  </oc-button>
+</template>
+<script lang="ts">
+import { computed } from 'vue'
+import { useGettext } from 'vue3-gettext'
+import { ref } from 'vue'
+import { watch } from 'vue'
+
+export default {
+  props: {
+    notificationCount: {
+      type: Number,
+      default: 0
+    }
+  },
+  setup(props) {
+    const { $gettext } = useGettext()
+    const notificationsLabel = computed(() => $gettext('Notifications'))
+    const notificationCountLabel = ref(`${props.notificationCount}`)
+    const animate = ref(false)
+
+    const updateNotificationLabel = () => {
+      if (props.notificationCount > 99) {
+        notificationCountLabel.value = '99+'
+      } else {
+        notificationCountLabel.value = `${props.notificationCount}`
+      }
+    }
```

Comment:
I think you can make your life much easier by just creating a computed prop for the notification label that reads `props.notificationCount` (and adds `+` if needed). It should handle all the reactive stuff automatically (if I'm not overseeing something 😅 ).

Reply:
oh yeah right, i think i had an 'brett vorm kopf' :D